### ISSUE TITLE
Please make it possible to use the right-ctrl key again

### ISSUE THREAD

**Issue by LaraSQP** [Oceanware/TameVisualStudioEditorToolTips#1]
Loved this little extension but, alas, I am left handed and can no longer use it after the latest update.

Could you please "right it" again?

Thank you so much for taking the time to develop all these extensions and making them available for free.

(I have tried building from source but, my friend, your setup is a nightmare)

**Comment by kdawg1406** [comment]
@LaraSQP apologies, I'll update it now.

Extensions can be challenging and with the MASSIVE changes to extensibility, it can be trying to figure out and properly set up your environment to compile them.

Give me 10 minutes.

**Comment by LaraSQP** [comment]
You are a god, sir.

**Comment by kdawg1406** [comment]
@LaraSQP it's live now.  Very sorry for non-sense.

Best to you,

Karl

**Comment by LaraSQP** [comment]
Got it installed and running. I'm good to go. Again, thank you and good karma to you.

You might want to know that the left-ctrl no longer seems to work. This does not affect me but this might be unintended and, perhaps, a problem for righties.

**Comment by kdawg1406** [comment]
@LaraSQP that's a 2:00 am typo!  Thank you for your outstanding testing and bug reporting.  Yes, I just shipped it w/o testing.  Ooops.    The code has a run super fast where the key was being tested.  So I reordered them to short-circuit fast.  I just messed up the moving of the keyboard state tests.

The new version is live now.

BTW:  Use the SHIFT key when trying to show the tooltip.  You won't get the clickable UI that the CTRL key can trigger.  SHIFT is a cleaner key (SHIFT was a suggestion from another happy user.)

Best to you,

Karl
